tests/python-gpu/test_from_cudf.py

```diff
@@ -382,3 +382,20 @@ def test_from_cudf_iter(enable_categorical):
     predict = reg.predict(m)
     predict_with_it = reg_with_it.predict(m_it)
     np.testing.assert_allclose(predict_with_it, predict)
+
+
+def test_invalid_meta() -> None:
+    df = cudf.DataFrame({"f0": [0, 1, 2], "f1": [2, 3, 4], "y": [None, 1, 2]})
+    y = df["y"]
+    X = df.drop(["y"], axis=1)
+    with pytest.raises(ValueError, match="Missing value"):
+        xgb.DMatrix(X, y)
+    with pytest.raises(ValueError, match="Missing value"):
+        xgb.QuantileDMatrix(X, y)
```

Comment:
Do we have a guide for a workaround in this use case? Is it enough to make a copy of `X` and `y`? If so, maybe we should add an example for that?

Reply:
Ah I just missed the `None` in the `y` column. So if we remove `None` from the example, you won't get an error?

Reply:
Correct.